src/plugins/quickdump/quickdump.c

```diff
@@ -394,7 +394,7 @@ int elektraQuickdumpSet (Plugin * handle, KeySet * returned, Key * parentKey)
 	kdb_unsigned_long_long_t magic = htobe64 (MAGIC_NUMBER_V3);
 	if (fwrite (&magic, sizeof (kdb_unsigned_long_long_t), 1, file) < 1)
 	{
-		fclose (file);
+		if (file != stdout) fclose (file);
```

Comment:
```suggestion
		closeFile (file);
```

I'm not a fan of `if`s without braces (`{}`), but my main problem here is that it is too easy to forget the adding the `if` and it's a lot of repetition.

I'd create a `static inline` function at the top of the file and just call that instead of `stdout`

```c
static inline void closeFile (FILE * file) {
    if (file != stdout) {
        fclose (file);
    }
}
```

Reply:
> I'm not a fan of ifs without braces ({})

We already have an issue about that:  #3734

> I'd create a static inline function at the top of the file and just call that instead of stdout

This fix was applied now, but there seems to be a broken link error now. Maybe it is only something temporary.

Otherwise looks good to me.